config/routes.rb

```diff
@@ -28,7 +28,7 @@
     resources :advisers, only: :show
   end
 
-  namespace :dashboard do
+  namespace :self_service, path: 'selfservice' do
```

Comment:
I figured that was what MAS would want. It's been updated as per rails default self_service